samply/src/windows/mod.rs

```diff
@@ -490,11 +490,17 @@ impl ProfileContext {
         // Virtualised ARM64 Windows crashes out on PROFILE tracing, and that's what I'm developing
         // on, so these are hacky args to get me a useful profile that I can work with.
         xperf.arg("-on");
-        //xperf.arg("PROC_THREAD+LOADER+PROFILE");
-        xperf.arg("PROC_THREAD+LOADER+CSWITCH+SYSCALL+VIRT_ALLOC+OB_HANDLE");
+        if self.arch != "aarch64" {
+            xperf.arg("PROC_THREAD+LOADER+PROFILE");
+        } else {
+            xperf.arg("PROC_THREAD+LOADER+CSWITCH+SYSCALL+VIRT_ALLOC+OB_HANDLE");
+        }
         xperf.arg("-stackwalk");
-        //xperf.arg("profile");
-        xperf.arg("VirtualAlloc+VirtualFree+HandleCreate+HandleClose");
+        if self.arch != "aarch64" {
+            xperf.arg("profile");
```

Comment:
This needs to be "profile,cswitch" so that we can get off-cpu stacks and so that we don't run into https://github.com/jrmuizel/etw-profiler/issues/9